support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -68,8 +68,8 @@ case object Contribution extends Product {
   lazy val ratePlans: Map[TouchPointEnvironment, List[ProductRatePlan[Contribution.type]]] =
     Map(
       PROD -> List(
-        ProductRatePlan("2c92a0fb4edd70c8014edeaa4eae220a", Monthly, NoFulfilmentOptions, NoProductOptions, "Monthly Contribution"),
-        ProductRatePlan("2c92a0fb4edd70c8014edeaa4e972204", Annual, NoFulfilmentOptions, NoProductOptions, "Annual Contribution"),
+        ProductRatePlan("2c92a0fc5aacfadd015ad24db4ff5e97", Monthly, NoFulfilmentOptions, NoProductOptions, "Monthly Contribution"),
```

Comment:
I discovered that we have had the wrong product rate plan ids specified here for a very long time. They are not actually used in the app though - contributions uses values from config. It would be good to standardise on these at some point.